src/main/java/racinggame/view/OutputView.java

```diff
@@ -0,0 +1,52 @@
+package racinggame.view;
+
+import racinggame.domain.Car;
+import racinggame.domain.RacingGame;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class OutputView {
+    private static int maxPosition = 0;
+    static List<String> winners = new ArrayList<>();
+
+	static void printRoundResult(RacingGame racingGame) {
+		for (String result : racingGame.play()) {
+			System.out.println(result);
+		}
+	}
+
+	public static void printResults(int count, RacingGame racingGame) {
+		System.out.println("실행 결과");
+
+		for (int i = 0; i < count; i++) {
+			printRoundResult(racingGame);
+			System.out.println();
+		}
+
+        for (Car car : racingGame.getCars()) {
+            if (maxPosition < car.getPosition()) {
+                maxPosition = car.getPosition();
+            }
+        }
+	}
+
+    public static void printWinners(RacingGame racingGame) {
+        for (Car car : racingGame.getCars()) {
+            if (car.getPosition() == maxPosition) {
+                winners.add(car.getName());
+            }
+        }
```

Comment:
우승자를 구하는 로직은 핵심 도메인 로직으로 보인다.
우승자를 구하기 위한 자동차 목록을 가지고 있는 RacingGame이 담당하면 어떨까?


```suggestion
        List<Car> winners = racingGame.findWinners();
```